### PR TITLE
Component/landingpage

### PR DESCRIPTION
**Pull request by tasneembhiri**
### content
- [x]  navbar
- [x]  I added two picture
- [x]  I added introduction 
- [x]  I added Book Appointment Button (when on click this button move to guideline page)
- [x] I added first style for desktop 
- [x] make it responsive

### Testing
on terminal use the following commands:
- git checkout component/landingpage
- git pull origin  component/landingpage
- npm run client

### CODE REVIEW

**Comment by okjake** [comment]
Nice start! I think you should take a look at the Grid component, it should help with the responsive layout

File: client/src/components/layouts/LandingPage/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Typography, Button, Grid } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import image from './image.svg';
+import footer from './footer.svg';
```
Comment (sara219) [inline client/src/components/layouts/LandingPage/index.js]: Add all the image into a dir called **(assets)**

File: client/src/components/layouts/LandingPage/style.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'space-around',
+    },
+  },
+  button: {
+    marginTop: '20px',
+    marginLeft: '-50px',
+    [theme.breakpoints.up(950 + theme.spacing(2) * 2)]: {
+      marginLeft: '-280px',
```
Comment (sara219) [inline client/src/components/layouts/LandingPage/style.js]: So much margin-left with a negative number!
maybe you can use top and right to handle the button position 

```
top: '2rem',
right: '18rem',
```

File: client/src/components/layouts/LandingPage/style.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  landing: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+
+    [theme.breakpoints.up(950 + theme.spacing(2) * 2)]: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'space-around',
```
Comment (sara219) [inline client/src/components/layouts/LandingPage/style.js]: For the mobile view, can you make the description and button at the centre

File: client/src/components/layouts/LandingPage/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+              To prevent the spread of COVID-19 and reduce the potential risk of
+              exposure to our staff and visitors, all visitors must complete an
+              intake form before booking an appointment.To begin this quick and
+              easy process, click Book Appointment below.
+            </Typography>
+            <Link to="/guideline" className={classes.link}>
+              <Button
+                variant="contained"
+                color="primary"
+                className={classes.button}
```
Comment (sara219) [inline client/src/components/layouts/LandingPage/index.js]: write a descriptive className

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by sara219 — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]
:woman_dancing: :partying_face:

**Review by Jood80 — APPROVED** [review]

**Pull request merged**